docs/criteria/accelerators/beta.md

```diff
@@ -1,5 +1,22 @@
 # Spice.ai OSS Data Accelerators - Beta Release Criteria
 
+## Contents
```

Comment:
This is a relatively short document... do we really need a contents that we'll have to keep up to date?